forms/keyword/file_search_form.go

```diff
@@ -0,0 +1,120 @@
+package forms
+
+import (
+	"mime/multipart"
+
+	. "go-crawler-challenge/forms"
+	"go-crawler-challenge/helpers"
+	"go-crawler-challenge/models"
+
+	"github.com/beego/beego/v2/client/orm"
+	"github.com/beego/beego/v2/core/logs"
+	"github.com/beego/beego/v2/core/validation"
+)
+
+type FileSearchForm struct {
+	File       multipart.File
+	FileHeader *multipart.FileHeader
+	User       *models.User `valid:"Required"`
+
+	keywordList []string
+}
+
+func (form *FileSearchForm) Valid(validation *validation.Validation) {
+	if form.File == nil {
+		err := validation.SetError("File", ValidationMessages["RequireFile"])
+		if err == nil {
+			logs.Warning("Set validation error failed")
```

Comment:
We are repeating this `Set validation error failed` multiple times, shall we cove it into constant variable and reuse that?